### PR TITLE
Add colorlog as runtime dependency

### PR DESCRIPTION
**Pull request by saraedum**
supersedes #4.

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -40,6 +40,7 @@ requirements:
     - six
     - beniget >=0.2.0
     - xsimd =7.2.2  # [not win]
+    - colorlog  # [py3k]
```
Comment (serge-sans-paille) [inline recipe/meta.yaml]: colorlog not supported anymore on py27 ? not that it matters, just out of curiosity.

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender

**Comment by saraedum** [comment]
@serge-sans-paille should colorlog still be installed at runtime? If so, I think this can be merged.

**Review by serge-sans-paille — APPROVED** [review]

**Pull request merged**